samtranslator/translator/translator.py

```diff
@@ -268,6 +276,101 @@ def _get_resources_to_iterate(
 
         return functions + statemachines + apis + others + connectors
 
+    @staticmethod
+    def _update_resources(resources: Dict[str, Any], connectors_list: List[SamConnector]) -> None:
+        for connector in connectors_list:
+            resources.update(connector.to_dict())
+
+    @staticmethod
+    def _delete_connectors_attribute(resources: Dict[str, Any]) -> None:
+        for source_logical_id, resource in resources.items():
+            if "Connectors" not in resource:
+                continue
+            del resources[source_logical_id]["Connectors"]
```

Comment:
Note sure if this works:

```suggestion
            del resource["Connectors"]
```